packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -192,56 +198,106 @@ Coming in the next release
 - *Behavior, dialed, color, artifact, center, license* (for both verb and noun), *-ize* endings, *program* (for all meanings)
 
 #### Bold and italics
-Coming in the next release
+
+##### Bold 
+**Do** use bold type sparingly, to draw attention to the most important word, number, or short phrase in your copy. 
+- Bold type can help a user scan the page more effectively by making landmarks more prominent.
+
+**Do** consider using bold type to refer to UI elements in instructional content. E.g.
+- *Select the location from **Region/State**.*
+- *Select **Continue** to review your order.*
+
+**Don't** use bold multiple times in body copy or for a whole sentence/paragraph. It dilutes its effect.
+- Only **bold the most important information the user needs to retain** and deploy it only if user testing flags a need. 
+- Consider if putting an important sentence on a new line will do a better job of making it stand out than making it bold. 
+
+##### Italics
+**Avoid** using italic type wherever possible. 
+- If you need to draw attention to a word, use bold type.
+- If you need to quote a phrase, use quotation marks "like this." 
 
 #### Capitalization
 **Do** capitalize proper nouns – products, people, places.
-- If a brand or product name uses a mix of upper and lower case (TwiML, eBay) use the name's convention.
 - Products include features, programs, or systems that Twilio has given a specific name to.
+- If a brand or product name uses a mix of upper and lower case (TwiML, eBay) use the name's convention.
 - People are named individuals or identifiable roles.
   - *Kris Brunner, Chief Engineer*
-  - *Contact your System Administrator*
 
-**Don't** capitalize general terms (*email, contact center, platform*)
+**Don't** capitalize general terms (*email, contact center, platform, phone number*)
 - Tip: If it's something Twilio has created or owns, it takes a capital.
 
 #### Code snippets
 Coming in the next release
 
-#### Dates
+#### Currency
 Coming in the next release
 
+#### Dates
+**Do** use American formatting as your default way to represent dates.
+- *MM/DD/YYYY* 
+- *Day, Month DD, YYYY* e.g., Sunday, May 17, 2020
```

Comment:
Is one of these recommended over the other?

Reply:
Noted :)